setup.py

```diff
@@ -4,13 +4,13 @@
 """
 
 import os
+import sys
 
-try:
-    from setuptools import setup
-except ImportError:
-    from distutils.core import setup
+from setuptools import setup
 
-import versioneer
+sys.path.append(os.path.dirname(__file__))
```

Comment:
Can you add a comment about why this is needed? (eg copy what I did for dask-geopandas: https://github.com/geopandas/dask-geopandas/blob/5b49377352658e15c34cb38a43b18fcc8f833ccb/setup.py#L6-L9)